games/sdl_sopwith/Portfile

```diff
@@ -23,10 +23,16 @@ checksums           md5     9527c13fd463731d0e189deb24eb39f8 \
                     sha256  afef3efb49994aec6a37185bde97af46d7231a7bf18dfbc6d3c87251f3f9dd46 \
                     size    1046380
 
-depends_build       port:pkgconfig
+depends_build       port:pkgconfig \
+                    port:automake
```

Comment:
Automake shouldn't be needed if only patching Makefile.in.

Reply:
Yep, removed.